legacy/conbenchlegacy/_version.py

```diff
@@ -0,0 +1,2 @@
+# Do not touch this autogenerated file
+__version__ = "2023.4.10"
```

Comment:
IMO we should not stick this package on PyPI; users are mostly installing from GitHub anyway. But maybe this version number should be incremented?

Reply:
Agreed we shouldn't put this on PyPI. So the version number doesn't matter. Could you remove the comment in this file?